synology-letsencrypt.sh

```diff
@@ -21,6 +21,9 @@ source "$cert_id_path"
 
 ## install hook
 archive_path="/usr/syno/etc/certificate/_archive/$cert_id"
+if [[ ! -s $archive_path ]]; then
```

Comment:
would it not be more appropriate to test with '-d' ?

Reply:
Yes you right, just a fast C/P from my side.

Reply:
I have squashed the commits :D